requirements.txt

```diff
@@ -0,0 +1,5 @@
+minerl=="^0.3.7"
```

Comment:
what does the ^ symbol do here? I get `ERROR: No matching distribution found for minerl=="^0.3.7"` when doing `pip install -r requirements.txt`

Reply:
yeah... sorry about that; this is fixed